core-bundle/src/Resources/contao/forms/Form.php

```diff
@@ -405,18 +405,18 @@ protected function processFormData($arrSubmitted, $arrLabels, $arrFields)
 			$email->fromName = $GLOBALS['TL_ADMIN_NAME'];
 
 			// Get the "reply to" address
-			if (!empty(Input::post('email', true)))
+			if (!empty($arrSubmitted['email']))
 			{
-				$replyTo = Input::post('email', true);
+				$replyTo = $arrSubmitted['email'];
```

Comment:
Do we need to decode entities here? As the second parameter of `Input::post()` enabled entity decoding.

Reply:
True. We could still use `Input::post('email', true)` etc. here, as long as we use `$arrSubmitted` for the `empty()` checks I suppose.

Reply:
I opted for `StringUtil::decodeEntities` instead.